start.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+#
+# Start script for chd-order-api
+
+APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    source ~/.chs_env/private_env
+    source ~/.chs_env/global_env
+    source ~/.chs_env/chd-order-api/env
+
+# FIXME replace with correct variable name
```

Comment:
the comments in this line and line 31 should be removed?

Reply:
true, will do that